docs/pages/blog/mui-x-v6.md

```diff
@@ -0,0 +1,188 @@
+The ApiRef enables developers to implement a whole new range of customizations that rely on programmatic control of the Grid's features.
+
+```tsx
+function CustomDataGrid(props) {
+  const apiRef = useGridApiRef();
+
+  return (
+    <div>
+      <Button onClick={() => apiRef.current.setPage(0)}>
+        Go to page first page
+      </Button>
+      <DataGrid apiRef={apiRef} {...other} />
+    </div>
+  );
+}
+```
+
+#### [Improved column menu](/x/react-data-grid/column-menu/)
+
+Another significant step in terms of customization but also usability; the v6 column menu now provides support for icons, menu groups, custom items and actions, and more.
+We've redesigned this sub-component to make it as extensible as possible.
+
+This is part of an overarching effort throughout the life cycle of v6 to refactor all Data Grid panels.
+
+<img src="/static/blog/mui-x-v6/column-menu-custom-action.png" style="maxWidth: 692px; margin-bottom: 24px;" loading="lazy" alt="Column menu custom action" />
+
+#### [Cell selection](/x/react-data-grid/cell-selection/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan)
+
+Select a cell or group of cells like in an Excel sheet.
+Cell selection is a powerful and flexible way to select data in the Data Grid.
+It's also the base for bulk editing and clipboard importing (coming soon).
+
+<video style="margin-bottom: 5px; width: 100%;" autoplay muted loop playsinline >
+  <source src="/static/blog/mui-x-v6/cell-selection.mp4" type="video/mp4" />
+</video>
+
+#### Stable [Aggregation](/x/react-data-grid/aggregation/) [<span class="plan-premium"></span>](/x/introduction/licensing/#premium-plan) and [Row pinning](/x/react-data-grid/row-pinning/) [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan)
+
+These two features have been previously released in a minor v5 version under the `experimentalFeatures` flag.
+Now they are officially stable.
+
+### Date and Time Pickers
+
+#### [Overhaul in the documentation](/x/react-date-pickers/)
+
+The Date and Time Pickers documentation has drastically improved during pre-release, and it now features a more comprehensible navigation structure and many new examples.
+
+#### [Improved layout customization](/x/react-date-pickers/custom-layout/)
+
+Combining the slots concept with the grid layout, you can now rearrange, extend, and customize most of the sub-components used in the Pickers UI.
+
+```tsx
+function MyCustomLayout(props) {
+  const { toolbar, tabs, content, actionBar } = usePickerLayout(props);
+
+  // Put the action bar before the content
+  return (
+    <PickersLayoutRoot className={pickersLayout.root} ownerState={props}>
+      {toolbar}
+      {actionBar}
+      <PickersLayoutContentWrapper className={pickersLayout.contentWrapper}>
+        {tabs}
+        {content}
+      </PickersLayoutContentWrapper>
+    </PickersLayoutRoot>
+  );
+}
+export default function CustomStaticDatePicker() {
```

Comment:
```suggestion

export default function CustomStaticDatePicker() {
```

Reply:
I removed this empty line to avoid scrolling in this demo box (which impacts the page scrolling).

Reply:
This was added in https://github.com/mui/material-ui/pull/33100 because it's support hard to read when the code takes all the viewport. We tried to increase it a bit from 350 to 400 in https://github.com/mui/material-ui/pull/33100#discussion_r929066849. Maybe 450 would be even better?